### DISCUSSION TITLE
how to get spawners to work in a plot

### DISCUSSION THREAD

**Discussion by tvgblade** [TownyAdvanced/Towny#5850] [Q&A]
spawners are not working in any towny plots even if mobs are turned on but as soon as the spawner is put in wilderness it works? how do I fix this?

**Comment by LlmDl** [comment]
Stand on the spawner and use `/plot perm` and show me the screenshot of the output.

**Reply by tvgblade** [reply]
![image](https://user-images.githubusercontent.com/103161373/162343474-3fa32870-8bc6-4d98-a4e4-b623cab3d33b.png)

**Reply by LlmDl** [reply]
I would file a bug report instead of a discussion. I suspect its not Towny affecting your server though. Either mob-spawning settings or another plugin.
